Docs: removes cookie secure example from Enterprise tab (#1172)
pomerium/documentation: content/docs/reference/cookies.mdx

</TabItem>
<TabItem value="Enterprise" label="Enterprise">

> As of v0.25, we've removed support for the **Cookie Secure** setting in the Enterprise Console (in the Console, this setting was called **Javascript Security**). Now, this setting is always enabled by default.

Reviewer: As far as I can tell, the part "Now, this setting is always enabled by default" is not correct. I think the UI was removed but not the backend logic, so Console may continue to control this setting (using whatever value is still in the database).

```suggestion
As of v0.25, we've removed the **Cookie Secure** setting from the Enterprise Console (in the Console, this setting was called **HTTPS only**).
```
